packages/vue/src/components/molecules/SfAddressPicker/SfAddressPicker.vue

```diff
@@ -12,14 +12,14 @@ Vue.component("SfAddress", SfAddress);
 export default {
   name: "SfAddressPicker",
   props: {
-    selected: {
+    value: {
```

Comment:
Sounds like a breaking change (or maybe earlier it was like that?) - if not, you could use https://vuejs.org/v2/api/#model